store/sqlstore/plugin_store.go

```diff
@@ -9,6 +9,7 @@ import (
 	"fmt"
 	"net/http"
 
+	sq "github.com/Masterminds/squirrel"
```

Comment:
Nit: this should be in a separate import group.

Reply:
Fixed, thank you for the review @agnivade ! :slightly_smiling_face: